### ISSUE TITLE
Ogg/Theora plugin doesn't work on macOS/Windows

### ISSUE THREAD

**Issue by totebo** [gideros/gideros#354]
The plugin is not found on Mac export or Gideros player; this code returns theora_is_loaded as false:
```
	local Theora
	local loadTheora =
		function()
			Theora = require "Theora"
		end

	local theora_is_loaded = pcall( loadTheora )
```

On PC theora_is_loaded returns true, but this code:

```
videoSprite = videoSource:play()
stage:addChild( videoSprite )
```

Causes this error:

`bad argument #1 to 'addChild' (Sprite expected, got table)`

**Comment by hgy29** [comment]
`videoSource:play()` will return a `Channel` (same as `Sound:play()`). What you want is:
```
videoSprite = videoSource:play():getVideo()
stage:addChild( videoSprite )
```

**Comment by hgy29** [comment]
Fixed for mac in c24f7939764499a38c1d7fcdd47ab306e4588f87
